agent/engine/docker_task_engine.go

```diff
@@ -741,16 +776,23 @@ func (engine *DockerTaskEngine) pullAndUpdateContainerReference(task *api.Task,
 	if container.IsInternal() {
 		return metadata
 	}
+	PullSucceeded := metadata.Error == nil
```

Comment:
why is this `PullSucceeded` and not `pullSucceeded`?

Reply:
Yes, it should be `pullSucceeded`.